apps/zadmin/tests/test_tasks.py

```diff
@@ -43,20 +52,20 @@ def make_langpack(version):
               <Description about="urn:mozilla:install-manifest"
                            em:id="langpack-de-DE@firefox.mozilla.org"
                            em:name="Foo Language Pack"
-                           em:version="%(version)s"
+                           em:version="{0}"
                            em:type="8"
                            em:creator="mozilla.org">
 
                 <em:targetApplication>
                   <Description>
-                    <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
-                    <em:minVersion>%(version)s</em:minVersion>
-                    <em:maxVersion>%(version)s.*</em:maxVersion>
+                    <em:id>{{ec8030f7-c20a-464f-9b0e-13a3a9e97384}}</em:id>
+                    <em:minVersion>{0}</em:minVersion>
+                    <em:maxVersion>{1}</em:maxVersion>
                   </Description>
                 </em:targetApplication>
               </Description>
             </RDF>
-        """ % {'version': version}
+        """.format(*versions)
```

Comment:
You changed the code to use `.format()` (which is better), but you used `%` many time above for the logs.